test/utils/test_pauli_twirling.py

```diff
@@ -37,6 +49,136 @@ def test_twirl_gates_cnot(self):
             circuit.append(d, [1])
             self.assertTrue(Operator(circuit).equiv(operator))
 
+    def test_twirl_gates_cy(self):
+        """Test twirling CY"""
+        twirl_gates = TWIRL_GATES["cy"]
```

Comment:
Add a test assertion that asserts that `twirl_gates` has the expected length (16?)

Reply:
Is this necessary? Nothing wrong if the twirl gate set is less or great than 16 right?

Reply:
Yeah that makes sense. Just FYI the twirl set is 16 for all gates except a couple like CSH gate (which is length 4) .